api/controllers/environment.go

```diff
@@ -88,8 +99,31 @@ func (e *EnvironmentController) UpdateEnvironment(c *fireball.Context) (fireball
 
 	if err := req.Validate(); err != nil {
 		return nil, errors.New(errors.InvalidRequest, err)
-
 	}
 
 	return createJob(e.JobStore, job.UpdateEnvironmentJob, req)
 }
+
+func (e *EnvironmentController) LinkEnvironment(c *fireball.Context) (fireball.Response, error) {
```

Comment:
This should be handled in a separate branch/pr

Reply:
agreed ✅